pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -856,11 +857,71 @@
     ::
         %channel
       (handle-request:by-channel secure authenticated address request)
+    ::
+        %scry
+      (handle-scry authenticated address request(url suburl))
     ::
         %four-oh-four
       %^  return-static-data-on-duct  404  'text/html'
       (error-page 404 authenticated url.request ~)
     ==
+  ::  +handle-scry: respond with scry result, 404 or 500
```

Comment:
Can you describe some use-cases this would serve? So far I think I can imagine it being used to serve a `graph` as an RSS feed.